### DISCUSSION TITLE
jquery security vulnerability detected after audit

### DISCUSSION THREAD

**Discussion by SimonWotton** [jquery/jquery#5319] [Q&A] [answered]
This might be totally clear to you guys, unfortunately I have no experience with this, so please help me understand and fix this problem.

I had an audit here at work and they found that a "webserver" on a device located in our network uses jquery 1.2
This is apparently a security issue and the audit people want me to update this to at least 3.5.0. I thought this is just a simple update thing like if you need a new java-version - just install update, done. After some googling it seems that this is not the case....

As I understand, I need to find the software that uses jquery 1.2 and then somehow upgrade and test it manually. Unfortunately, I have no clue how to do this and the device in question is a machine that I will get in trouble for messing up if anything goes wrong. I don't know what jquery is used for, if it is crucial for running the machine, etc. The audit people, however, insist on this being fixed.

So please let me know - what must be done here and how can I get it done with basically zero knowledge of this kind of thing. I do have many years if IT under my belt, but in the field, not in development :-(

Please help me - and thank you for your time!

**Comment by mgol** [comment, marked as answer]
There's no simple answer here. Breaking jQuery updates have upgrade guides at https://jquery.com/upgrade-guide/, using jQuery Migrate is also recommended to ease the upgrade.
